src/pages/Mint.tsx

```diff
@@ -1,238 +1,275 @@
-import React, { useContext, useEffect, useState } from 'react'
+import React, { useContext, useState } from 'react'
 import { useRecoilValueLoadable } from 'recoil'
 import { charitiesState, ICharitiesData } from '../state/charities'
-import { mintThing } from '../state/near'
+import { mintItem } from '../state/near'
 
 import loader from 'url:../assets/loader.gif'
 import { MintbaseContext } from '../contexts/mintbase'
-import { toast } from 'react-toastify'
-
-const FilePreview: React.FC<{ file: any }> = ({ file }) => {
-  const url = URL.createObjectURL(file)
+import { FileField } from '../components/FileUploadField'
+import { FormField } from '../components/FormField'
+import { IPriceSplits, PriceSplitsField } from '../components/PriceSplitsField'
 
+const Mint: React.FC = () => {
   return (
-    <article
-      tabIndex={0}
-      className="group hasImage w-60 h-60 rounded-md focus:outline-none focus:shadow-outline bg-gray-100 cursor-pointer relative text-transparent hover:text-white shadow-sm"
-    >
-      <img
-        alt={file.name}
-        className="img-preview w-full h-full sticky object-cover rounded-md bg-fixed"
-        src={url}
-      />
-    </article>
-  )
-}
+    <div className="w-full p-5 grid grid-flow-col-dense grid-cols-2 sm:grid-cols-2">
+      <div>
+        <h2 className="text-2xl">What does it mean to mint?</h2>
 
-const MintButton: React.FC<{ onClick?: () => void }> = ({ onClick }) => {
-  return (
-    <div
-      className="p-5"
-      onClick={() => {
-        onClick && onClick()
-      }}
-    >
-      <div className="relative ">
-        <div className="absolute bg-black top-0.5 -right-0.5 -bottom-0.5 left-0.5"></div>
-        <div
-          className="relative text-sm border cursor-pointer"
-          style={{
-            height: 38,
-            backgroundColor: 'rgb(190, 242, 100)',
-            borderColor: 'rgb(0, 0, 0)',
-          }}
-        >
-          <span className="absolute inset-0 flex items-center justify-center text-black tracking-wide uppercase RobotoMono-Bold_font__226og">
-            <span>&nbsp;Mint&nbsp;</span>
-          </span>
-        </div>
-      </div>{' '}
+        <span className="text-base">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
Let's put here something meaningful or remove

Reply:
I have added Lorem ipsum to have some placeholder for text. This form looks much better not on full screen. Will add some simple description

Reply:
Please notice comment in main chat